packages/create-liveblocks-app/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "create-liveblocks-app",
-  "version": "1.5.2",
+  "version": "1.5.3-test1",
```

Comment:
Side-note: I find these auto-bumps on PRs a little bit annoying. If you do multiple of them, they make merging back to main harder, especially when there are multiple PRs that are issuing test releases. Maybe we should change the job to not push those bump commits to the PR. Instead, it can create that commit, but not on a branch, tag that (dangling) commit, and publish it. Same thing, except the PR isn't polluted.

Reply:
I agree